.github/workflows/master.yml

```diff
@@ -17,19 +17,19 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
-      - uses: google-github-actions/auth@v0
+      - uses: google-github-actions/auth@v2
         with:
-          credentials_json: ${{ secrets.GCR_CREDENTIALS_JSON }}
+          credentials_json: ${{ secrets.DEV_GAR_KEY }}
```

Comment:
GAR key?

Reply:
Google Artifact Registry , its all the rage!